catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/AbstractFilterVisitor2.java

```diff
@@ -90,8 +96,7 @@ public void visitBinarySpatialType(VisitableElement<List<Object>> visitable) {
   @Override
   public void visitDistanceBufferType(VisitableElement<List<Object>> visitable) {
     traceName(visitable);
-    throw new UnsupportedOperationException("DistanceBufferType currently is not supported");
```

Comment:
✏️ If this ticket is done then we should close it out on Jira

https://codice.atlassian.net/browse/DDF-3829

Reply:
This only implements a subset of the missing functionality specified in that ticket. I left a comment referencing DDF-4373